File: backend/WorldBeauty/Telefone.java
package WorldBeauty;

public class Telefone {
    private String ddd;
    private String numero;

    Telefone(String ddd, String numero) {
        this.ddd = ddd;
        this.numero = numero;
    }

    public String GetDDD() {
        return this.ddd;
    }

    public String GetNumero() {
        return this.numero;
    }

    public String GetNumeroFormatado() {
        return this.ddd + " " + this.numero.substring(0, 6) + "-" + this.numero.substring(6, 12);
    }
}

File: world-beauty/src/WB-ATV1/src/Models/Telefone.ts
export default class Telefone {
    private ddd: string;
    private numero: string;

    constructor(ddd: string, numero: string) {
        this.ddd = ddd;
        this.numero = numero;
    }

    public get GetDDD(): string {
        return this.ddd;
    }

    public get GetNumero(): string {
        return this.numero;
    }

    public get GetNumeroFormatado(): string {
        return `(${this.ddd}) ${this.numero.slice(0, 6)}-${this.numero.slice(6, 12)}`;
    }
}